homeassistant/components/met_eireann/config_flow.py

```diff
@@ -0,0 +1,57 @@
+"""Config flow to configure Met Éireann component."""
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ELEVATION, CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+# pylint:disable=unused-import
+from .const import DOMAIN, HOME_LOCATION_NAME
+
+
+class MetEireannFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Met Eireann component."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Init MetEireannFlowHandler."""
+        self._errors = {}
```

Comment:
Don't store errors as an instance attribute. The form errors are local to each step method.